### PR TITLE
Rewritten CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by TheBusyBiscuit**
## Description

I also rewrote the CONTRIBUTING.md file now.

@WalshyDev Tell me what you think, whether I missed anything or whether something should be changed or added.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Very active community translators will have the option to become a "language moderator". 
+Language moderators are responsible for proof-reading any new translations for their designated language and correct it when they see a mistake.
+
+For more info on how or what to translate, check out our article on [How to translate Slimefun](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Translating-Slimefun).
+
+## 5. Wiki contributions
+Slimefun is a very large project and might be quite intimidating for new players.
+That's why good documentation is always nice and helpful.
+If you have played with Slimefun for a while and gotten yourself familiar with how things work, please consider contributing your experiences and knowledge to others via the wiki!
+It would help out a lot <3
```
Comment (WalshyDev) [inline .github/CONTRIBUTING.md:52]: do `:heart:`

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Contributing to Slimefun
+This document outlines various ways how you can help contribute to Slimefun and make this a bigger and better project.<br>
```
Comment (LinoxGH) [inline .github/CONTRIBUTING.md:2]: I feel like "help contribute to" part has some grammatical errors. I'd've used "help by contributing to" and "and make this " to "and making this"

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+All contributions must be inline with our [Code of Conduct](https://github.com/TheBusyBiscuit/Slimefun4/blob/master/.github/CODE_OF_CONDUCT.md) and [License](https://github.com/TheBusyBiscuit/Slimefun4/blob/master/LICENSE).
+Please also follow the templates for Issues and Pull Requests we provide.
+
+## 1. Issues: Bug Reports
+One of the foundations for good software is reliability. To facilitate this reliability, our community must work together to crush bugs that arise. 
+This of course requires good information and knowledge about ongoing bugs and issues though.
```
Comment (LinoxGH) [inline .github/CONTRIBUTING.md:8]: I think this line could use some punctuation too like "This, of course, requires..."

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+So if you have an idea on how to approach a known issue, feel free to make a [Pull Request](https://github.com/TheBusyBiscuit/Slimefun4/pulls) which fixes this bug.
+You can also comment on the existing Issue, proposing your idea or communicating that you wanna work on this.
+
+## 3. Pull Requests: Additions/Changes
+Slimefun is an Open-Source project and anyone is allowed to make changes or add content to this plugin!
+
+Please visit our [Discord Server](https://github.com/TheBusyBiscuit/Slimefun4#discord) and share your ideas first, we hate to reject changes because the community disagrees.<br>
+So communicating your intended changes before-hand will ensure that you don't put too much work into something that might get rejected.<br>
+We also have a suggestions section in our Discord Server too, you can check out `#approved` for any suggestions the community has voted for. 
+It is a great place to start looking for ideas on what to add or change.
```
Comment (WalshyDev) [inline .github/CONTRIBUTING.md:32]: In this section we talk about sharing ideas and they're in approved when the community votes but it doesn't explicitly state what channel to put ideas in. Even though it's simple if they went into the server we should still just mention #suggestions here

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+Also consider making an addon for your additions when they get too large, check out our [Developer Guide](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Developer-Guide).
```
Comment (LinoxGH) [inline .github/CONTRIBUTING.md:34]: Maybe you should also add "or when they are very specialized frpm the rest of the Slimefun"

**Review by WalshyDev — COMMENTED** [review]
Looking good.

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+## 4. Translations
+Another great way to contribute to Slimefun is by working on translations for the project.
+Slimefun's translation is available on [gitlocalize.com](https://gitlocalize.com/repo/3841).
+Just find a language you are fluent in and translate away. But make sure to submit a "Review Request" when you are done.
+One of our language moderators will review the changes and submit a Pull Request to the project for you.
+
+#### Language moderation
+Very active community translators will have the option to become a "language moderator".
```
Comment (LinoxGH) [inline .github/CONTRIBUTING.md:43]: I think you should capitalize the first letters of "language moderator". But this is a personal thought.

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+https://github.com/TheBusyBiscuit/Slimefun4/wiki/Expanding-the-Wiki
+
+## 6. Pull Requests: Code Quality
+Slimefun uses [sonarcloud.io](https://sonarcloud.io/dashboard?id=TheBusyBiscuit_Slimefun4) to monitor Code Quality.
+
+We always welcome quality improvements to the code and the "Code Smells" section on [sonarcloud.io](https://sonarcloud.io/dashboard?id=TheBusyBiscuit_Slimefun4) is a great place to start.
+But please keep in mind that some design patterns may not be changed too abruptly if an addon depends on them. 
+To prevent any accidents from happening, please contact us on our [Discord Server](https://github.com/TheBusyBiscuit/Slimefun4#discord) before-hand and state your intended changes.
+
+#### documentation
```
Comment (LinoxGH) [inline .github/CONTRIBUTING.md:64]: Capitalize the "d"?

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (TheBusyBiscuit) [reply .github/CONTRIBUTING.md:8]: A paranthesis would be wrong in this use case here.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (TheBusyBiscuit) [reply .github/CONTRIBUTING.md:2]: Help + infitive without the to is gramatically correct and the informal version of it.

File: .github/CONTRIBUTING.md (hunk 6 above)
Comment (TheBusyBiscuit) [reply .github/CONTRIBUTING.md:43]: Yeah that should probably be more consistent.

**Pull request merged**